packages/fetchai/connections/oef/connection.py

```diff
@@ -34,10 +34,12 @@
     Constraint as OEFConstraint,
     ConstraintExpr as OEFConstraintExpr,
     ConstraintType as OEFConstraintType,
+    Distance,
```

Comment:
Note, on our side we do not have a Distance model

Reply:
ok